src/services/kuma-api/Api.ts

```diff
@@ -0,0 +1,34 @@
+import { RestClient } from './RestClient'
+import Env from '@/services/env/Env'
```

Comment:
I think this one would be nicer as `import type` also, super nitpick tho.

Reply:
I’ve changed this.